Chess-Challenge/src/My Bot/MyBot.cs

```diff
@@ -68,16 +68,27 @@ public Move Think(Board board, Timer timer)
 
     Node alphaBetaSearch(Board board, int depth, int alpha, int beta, bool maximizing)
     {
-        Move[] moves = board.GetLegalMoves();
-        if (depth == 0 || moves.Length == 0)
+        // search capture moves first, big a/b pruning speed up. 
+        // use hashset for a faster way than simply sorting by IsCapture.
+        // tested against quicksort below, which is much slower
+        // List<Move> moves = board.GetLegalMoves().ToList();
+        // moves.Sort((a, b) => a.IsCapture.CompareTo(b.IsCapture));
+
+        List<Move> moves = board.GetLegalMoves(true).ToList();
```

Comment:
I saw in the API there is also `GetLegalMovesNonAlloc` which is a memory assignment optimization. A TODO for the future :D